### PR TITLE
doc: tweak doc build instructions

### PR DESCRIPTION
**Pull request by dbkinder**
There was a remaining reference to a doc/Makefile that needed
to be changed to CMakeLists.txt

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Review by nashif — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

File: doc/README.rst [hunk 1]
```diff
@@ -134,7 +134,7 @@ generation process output.
 The output from the Sphinx build is processed by the python script
 ``scripts/filter-known-issues.py`` together with a set of filter
 configuration files in the ``.known-issues/doc`` folder.  (This
-filtering is done as part of the ``doc/Makefile``.)
+filtering is done as part of the ``doc/CMakeLists.txt`` Cmake listfile.)
```
Comment (carlescufi) [inline doc/README.rst]: “CMake” not “Cmake”

**Review by carlescufi — CHANGES_REQUESTED** [review]
Approved by mistake. There’s a nit.

File: doc/README.rst (hunk 1 above)
Comment (nashif) [reply doc/README.rst]: lucky you :)

**Comment by dbkinder** [comment]
@carlescufi Cmake change to CMake.  Now get back to your newborn!

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
@dbkinder thanks! and... will do :)

**Pull request merged**